File: scripts/fix_eh.mjs
import fs from 'fs';
var nc = fs.readFileSync('src/registration/search/useMemberSearch.ts', 'utf8');
var lines = nc.split('\n');
if (lines[0].indexOf('import React') === -1) { lines.unshift(\"import React from 'react';\"); }
var lt = String.fromCharCode(60);
var gt = String.fromCharCode(62);
var typed = '(e: React.ChangeEvent' + lt + 'HTMLInputElement' + gt + ') => {';
for (var i = 0; i < lines.length; i++) {
  if (lines[i].trim() === '(e) => {') { lines[i] = lines[i].replace('(e) => {',typed); }
}
fs.writeFileSync('src/registration/search/useMemberSearch.ts', lines.join('\n'));
console.log('done');

File: scripts/fix_eh.py
c = open("src/registration/search/useMemberSearch.ts").read()
lines = c.split(chr(10))
qt = chr(39)
nt = "(e: { target: { value: string } }) => {"
for i in range(len(lines)):
    if lines[i].strip() == "(e) => {" and i > 150:
        lines[i] = lines[i].replace("(e) => {", nt)
if lines[0].find("import React") == -1:
    lines.insert(0, "import React from " + qt + "react" + qt + ";")
open("src/registration/search/useMemberSearch.ts", "w").write(chr(10).join(lines))
print("done")